src/components/navigation/nav.module.scss

```diff
@@ -0,0 +1,365 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,500,500i,700');
+@import '../../../styles/variables.scss';
+
+.navbar {
+  font-family: 'Roboto', sans-serif;
```

Comment:
font-family is not what used in design, use inter plz. as added as variables in variables.scss